features.html

```diff
@@ -154,6 +154,50 @@ <h2>List of Features</h2>
         <li>
             Sieve filters can be created to automatically move, copy, or delete messages based on specific criteria such as sender, subject, keywords, or recipient. This allows for efficient organization and management of incoming emails, saving time and improving productivity. Sieve filters can be easily edited or deleted as needed. For more information on how to create and manage Sieve filters, see <a href="email-filters.html">Email filters</a>.
         </li>
+        <hr>
+        <li>
+            JMAP (JSON Meta Application Protocol) support for faster, more efficient synchronization of emails, contacts, and calendars across devices.
```

Comment:
I know Cypht supports JMAP emails but are you sure about contacts and calendars?